pkg/installer/verify

```diff
@@ -58,6 +58,10 @@ do
    fi
 done
 
+# The following qemu test is commented out, and has been for a long time as of this writing.
```

Comment:
We are in the process of having a new interactive installer, this code has been commented out for a long time and we are printing `start VM test unavilable` always, that can confuse users. TBH, my vote is to just remove this piece of code... it can be brought back easily taking it from older commits....

Reply:
Sure, I will update it.

Reply:
Done